feat: Adding `cosmovisor version` command (#10103)
cosmos/cosmos-sdk: cosmovisor/README.md

The `DAEMON` specific code and operations (e.g. tendermint config, the application db, syncing blocks, etc.) all work as expected. The application binaries' directives such as command-line flags and environment variables also work as expected.

### Commands

> As Cosmovisor is meant to be used as a wrapper for a `Cosmos SDK` application, it does not support many commands. 
> 
> However, to determine the version of Cosmovisor, the following command can be run.

Reviewer: ```suggestion
Because Cosmovisor is meant to be used as a wrapper for a Cosmos SDK application, it does not require many commands. 

To determine the version of Cosmovisor, run the following command:
```

> However, to determine the version of Cosmovisor being run, the following command can be run.

Reviewer 2: ```suggestion
However, to determine the version of Cosmovisor, the following command can be run.
```

Author: 👍🏻

```
cosmovisor version
``` 

> This returns the version of the `Cosmos SDK` application as well as the version of Cosmovisor as shown below

Reviewer: ```suggestion
The output of the `cosmovisor version` command shows the version of the Cosmos SDK application and the version of Cosmovisor:

```

```
0.43.0-beta1-319-ge3aec1840

> Cosmovisor Version:  v0.1.0-85-g65baacac0

Reviewer 3: shouldn't we put the cosmovisor version first?

Author: Yea makes sense 👍🏻
